validations/transfer.test.ts

```diff
@@ -0,0 +1,138 @@
+      )
+
+      const quoteInParams = {
+        ...MOCK_QUOTE[config.quoteInMock],
+        address: wallet.address,
+      }
+
+      it('able to transfer fiat in for crypto', async () => {
+        const loginResult = await fiatConnectClient.login()
+        expect(loginResult.isOk).to.be.ok
+
+        const quoteInResponse = await fiatConnectClient.createQuoteIn(
+          quoteInParams,
+        )
+        expect(quoteInResponse.isOk).to.be.ok
+        const quoteId = quoteInResponse.unwrap().quote.quoteId
+
+        const addKycResult = await fiatConnectClient.addKyc(mockKYCInfo)
+        expect(addKycResult.isOk).to.be.true
+
+        const addAccountResult = await fiatConnectClient.addFiatAccount(
+          mockAccountData,
+        )
+        expect(addAccountResult.isOk).to.be.true
+        const fiatAccountId = addAccountResult.unwrap().fiatAccountId
+
+        const transferInParams = {
+          idempotencyKey: randomUUID(),
+          data: {
+            fiatAccountId: fiatAccountId,
+            quoteId: quoteId,
+          },
+        }
+
+        const transferInResponse = await fiatConnectClient.transferIn(
+          transferInParams,
+        )
+        expect(transferInResponse.isOk).to.be.ok
+        expect(transferInResponse.unwrap().transferStatus).to.be.oneOf(
+          Object.values(TransferStatus),
+        )
+      })
+    })
+  }
+
+  describe('/out', () => {
+    const wallet = new ethers.Wallet(config.testPrivateKey)
+
+    const fiatConnectClient = new FiatConnectClient(
+      {
+        baseUrl: config.baseUrl,
+        network: Network.Alfajores,
+        accountAddress: wallet.address,
+      },
+      (message: string) => wallet.signMessage(message),
+    )
+
+    const quoteOutParams = {
+      ...MOCK_QUOTE[config.quoteOutMock],
+      address: wallet.address,
+    }
+
+    it('able to transfer crypto in for fiat out', async () => {
+      const loginResult = await fiatConnectClient.login()
+      expect(loginResult.isOk).to.be.ok
+
+      const addKycResult = await fiatConnectClient.addKyc(mockKYCInfo)
+      expect(addKycResult.isOk).to.be.true
+
+      const addAccountResult = await fiatConnectClient.addFiatAccount(
+        mockAccountData,
+      )
+      expect(addAccountResult.isOk).to.be.true
+      const fiatAccountId = addAccountResult.unwrap().fiatAccountId
+
+      const quoteOutResponse = await fiatConnectClient.createQuoteOut(
+        quoteOutParams,
+      )
+      expect(quoteOutResponse.isOk).to.be.ok
+      const quoteOutId = quoteOutResponse.unwrap().quote.quoteId
+
+      const transferOutParams = {
+        idempotencyKey: randomUUID(),
+        data: {
+          fiatAccountId: fiatAccountId,
+          quoteId: quoteOutId,
+        },
+      }
+
+      const transferOutResponse = await fiatConnectClient.transferOut(
+        transferOutParams,
+      )
+      expect(transferOutResponse.isOk).to.be.ok
+      expect(transferOutResponse.unwrap().transferStatus).to.be.oneOf(
```

Comment:
I think we can be more specific and say it should be TransferStarted or TransferReadyForUserToSendCryptoFunds . See https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#42-transfers-out